client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -591,7 +619,17 @@ const summarise = (
       );
     } else {
       const unitType = getPlural(unitName, numUnits);
-      summary.push(t('label.packs-of-1', { numUnits, unitType }));
+      const dosesWord = t('label.doses-plural', { count: numUnitsWithDoses });
+      const displayDoses = displayInDoses
+        ? t('label.num-in-brackets', {
+            num: numUnitsWithDoses,
+            unitWord: dosesWord,
+          })
+        : '';
+
+      summary.push(
+        t('label.packs-of-1', { numUnits, unitType }) + ` ${displayDoses}`
```

Comment:
![Screenshot 2025-05-05 at 11 24 58 AM](https://github.com/user-attachments/assets/5e029294-435e-4fe4-a38a-8be6a69674af)


This sentence seems a bit non-sensical to me? Esp if what I care about is doses... surely just `1 dose` would be fine here, can open the details section if i care about packs/units?